### PR TITLE
Misc writing fixes

### PR DESCRIPTION
**Pull request by iglpdc**
Several minor fixes.

### CODE REVIEW

**Comment by iglpdc** [comment]
It's all pretty trivial, but if someone can take look would be nice.

File: 01-basics.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ minutes: 5
...
-> *   Understand the benefits of an automated version control system
-> *   Understand the basics of how Git works
+> *   Understand the benefits of an automated version control system.
+> *   Understand the basics of how Git works.
 
 We'll start by exploring how version control can be used
 to keep track of what one person did and when.
 Even if you aren't collaborating with other people,
-version control is much better for this than this:
+automated version control is much better than this:
```
Comment (willingc) [inline 01-basics.md]: You may wish to replace 'this' with a concrete thought (undocumented changes, unreproducible history, etc.)

File: 01-basics.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ If there aren't conflicts, you can even try to play two sets of changes onto the
 ![Multiple versions can be merged](fig/merge.svg)
 
 A version control system is a tool that keeps track of these changes for us and
-helps us version and merge our files. A version control system allows you to
+helps us version and merge our files. It allows you to
 decide which changes made up the next version, called a
```
Comment (willingc) [inline 01-basics.md]: make up...

File: 06-ignore.md [hunk 3]
```diff
@@ -88,8 +88,8 @@ $ git status
 nothing to commit, working directory clean
 ~~~
 
-As a bonus,
-using `.gitignore` helps us avoid accidentally adding files to the repository that we don't want.
+As a bonus, using `.gitignore` helps us avoid accidentally adding files to the
+repository that we don't want:
```
Comment (willingc) [inline 06-ignore.md]: ...want to track:

File: 09-conflict.md [hunk 4]
```diff
@@ -9,18 +9,16 @@ minutes: 15
...
-if we are working on a piece of software on both our laptop and a server in the lab,
-we could make different changes to each copy.
-Version control helps us manage these [conflicts](reference.html#conflicts)
-by giving us tools to [resolve](reference.html#resolve) overlapping changes.
-
-To see how we can resolve conflicts,
-we must first create one.
-The file `mars.txt` currently looks like this
-in both partners' copies of our `planets` repository:
+As soon as people can work in parallel, someone's going to step on someone
```
Comment (willingc) [inline 09-conflict.md]: ...parallel, it's likely that someone's...

File: index.md [hunk 5]
```diff
@@ -12,19 +12,20 @@ spend a lot of time waiting for the other to finish, but if they work
 on their own copies and email changes back and forth things will be
 lost, overwritten, or duplicated.
 
-A colleague suggests using [version control](reference.html#version-control) to manage their work.
-Version control is better than mailing files back and forth because:
+A colleague suggests using [version control](reference.html#version-control) to
+manage their work. Version control is better than mailing files back and forth:
+
+*   Nothing that is committed to version control is ever lost.  This means it
```
Comment (willingc) [inline index.md]: Replace "This means it..." with Version control's revert feature can be used like

**Comment by willingc** [comment]
@iglpdc This looks great! I've suggested a few minor things. None are critical, simply refinements :)

File: 01-basics.md (hunk 1 above)
Comment (iglpdc) [inline 01-basics.md]: Yes. The "this" refers to the comic below, so I changed to "this situation".

File: index.md (hunk 5 above)
Comment (iglpdc) [inline index.md]: I rewrote slightly the three bullet points. Now it reads better, I think.

**Comment by iglpdc** [comment]
Thanks for the review, @willingc !

**Comment by iglpdc** [comment]
I'm merging this.

**Pull request merged**